spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkPositionDeltaWrite.java

```diff
@@ -132,7 +132,9 @@ class SparkPositionDeltaWrite implements DeltaWrite, RequiresDistributionAndOrde
 
   @Override
   public Distribution requiredDistribution() {
-    return writeRequirements.distribution();
+    Distribution distribution = writeRequirements.distribution();
```

Comment:
Both `Distribution` and `SortOrder` implementation in Spark provide reasonable `toString()`.

```
23/09/26 17:00:55 INFO SparkWrite: Requesting 402653184 bytes advisory partition size for table testhive.default.table
23/09/26 17:00:55 INFO SparkWrite: Requesting ClusteredDistribution(bucket(8, c3)) as write distribution for table testhive.default.table
23/09/26 17:00:55 INFO SparkWrite: Requesting [bucket(8, c3) ASC NULLS FIRST, id ASC NULLS FIRST] as write ordering for table testhive.default.table
```